app/assets/stylesheets/views/_covid.scss

```diff
@@ -15,27 +15,40 @@ $c19-landing-page-header-background: govuk-colour("blue");
 
 .global-bar-present {
   @include govuk-media-query($from: desktop) {
-    .covid__page-header {
+    .covid__breadcrumb-wrapper {
```

Comment:
Do we think that these three renamed classes are still being used? Their parent is something called `.global-bar-present` but there doesn't seem to be 
`.global-bar-present` on the page or the app.

Reply:
I didn't change anything just to stay safe, but I also don't think `.global-bar-present` is still used, so I'm happy to remove this part. It's also used for browse pages, so I can also remove it from there.

Reply:
I dug into this and realised this for the global banner coming from static: https://docs.publishing.service.gov.uk/manual/global-banner.html#activate-the-global-banner It might be good to double check that these styles work if the global banner is there although I think it's been a bit waffy to test with the global banner in the past since it's in a separate repo 🤔

Reply:
I'll test it with a global banner enabled in static.

Reply:
I've just had a look at integration/coronavirus and it looks good to me with the global banner enabled 👍 I'm going to approve the PR but let me know if there was something else about the global banner I should be checking.

Reply:
That's all I think.